src/Ui/Input.elm

```diff
@@ -0,0 +1,54 @@
+module Ui.Input exposing (search)
+
+import Accessibility.Styled as Styled
+import Css
+import Extra.Styled as Styled
+import Html.Styled exposing (Attribute)
+import Html.Styled.Attributes exposing (css, property)
+import Html.Styled.Events
+import Json.Encode as JE
+import Rpx exposing (rpx)
+import Ui.Button
+import Ui.Color exposing (toCssColor)
+import Ui.Icon as Icon
+import Ui.Palette
+
+
+inputTextBaseStyle : Attribute msg
+inputTextBaseStyle =
+    css
+        [ Css.padding4 (Css.px 4) (Css.px 5) (Css.px 4) (Css.px 5)
```

Comment:
`padding2`? or even same padding everywhere?

Reply:
padding2 is good